### PR TITLE
Minor fixes for encumbrance granularity

### PR DESCRIPTION
**Pull request by DavidKeaton**
Just some minor fixes.
- Changed encumbrance values for wool, fur, leather, and Kevlar.
- Fixed spacing issues in display of this new tweak. (armor_layers)

-Davek

### CODE REVIEW

**Comment by KA101** [comment]
Granularity is fine, but the spacing on the = signs looked weird on mine: offset from the column.

**Comment by DavidKeaton** [comment]
will peek more at

**Comment by DavidKeaton** [comment]
fixed the spacing, was an issue where apparently not all (char *) are made equally in C++ (char *[4]? lol)

File: src/armor_layers.cpp [hunk 1]
```diff
@@ -175,12 +175,15 @@ void player::sort_armor()
             } else {
                 mvwprintz(w_sort_middle, cont_h - 12 + i, 2, c_ltgray, "%s:", armor_cat[i].c_str());
             }
-            mvwprintz(w_sort_middle, cont_h - 12 + i, middle_w - 16, c_ltgray, "%d+%d = ", armorenc,
-                      enc - armorenc);
+            true_enc = enc - armorenc;
+            char my_spaces[]    = "   ";    // lol
+            char *spaces        = (char*)&my_spaces;
```
Comment (BevapDin) [inline src/armor_layers.cpp]: `&my_spaces` is a `char**`, not a `char*` not at all. Btw. why not `const char*` and `const char my_spaces[]`.

This example complies fine for me:
```C++
    static const char my_space[] = "   ";
    const char *x = my_space;
    x++;
```

And C++ is *not* weird. Humans are weird, they say "or", but they mean "xor".

Comment (DavidKeaton) [inline src/armor_layers.cpp]: C++ is weird to me coming from ASM & C, lol. Also, that was an overlook on my part, as far as char** is concerned, I just simply wasn't getting the correct pointer's pointer. Still get fumbled sometimes on pointer pointers, as far as when to grab them.

**Pull request merged**